### PR TITLE
Tweaked some code per ESLint best practices

### PR DESCRIPTION
**Pull request by pdehaan**

### CODE REVIEW

File: index.js [hunk 1]
```diff
@@ -1,12 +1,17 @@
-var lists = [];
+'use strict';
+
 var fs = require('fs');
-var pathToLists = __dirname + '/spam_lists/';
+var path = require('path');
+
+var pathToLists = path.join(__dirname, 'spam_lists');
```
Comment (pdehaan) [inline index.js:6]: Use `path.join()` or `path.resolve()` instead of + to create paths

File: index.js [hunk 2]
```diff
@@ -1,12 +1,17 @@
...
+var lists = [];
+
 loadLists();
 
 function loadLists() {
 	fs.readdir(pathToLists, function(err, files) {
 		if(err) {
-			throw "could not find spam_lists directory";
+			throw new Error('could not find spam_lists directory');
```
Comment (pdehaan) [inline index.js:14]: Throw `Errors`, not strings, bro...
Plus, single quotes, for consistency.

File: package.json [hunk 3]
```diff
@@ -1,22 +1,25 @@
...
-  "license": "MIT"
+  "license": "MIT",
+  "main": "index.js",
+  "repository": {
+    "type": "git",
+    "url": "git://github.com/adityabansod/known-spam-emails.git"
+  },
+  "scripts": {
+    "test": "node test.js"
+  }
```
Comment (pdehaan) [inline package.json:24]: Sorry, i ran `fixpack` on this and I think it sorted some attributes.

**Comment by adityabansod** [comment]
:heart:

**Comment by pdehaan** [comment]
Look at us all remote pair programming! :swoon:

**Pull request merged**